versions/2.0/schema/digitized_audioopticaldisc.json

```diff
@@ -36,16 +36,27 @@
           "$ref": "fields#technicalFilename"
         },
         "extension": {
-          "$ref": "fields#technicalExtensionForAudio"
+          "enum": [
+            "bin",
+            "BIN",
+            "WAV",
+            "wav"
+          ]
         },
         "dateCreated": {
           "$ref": "fields#technicalDateCreated"
         },
         "fileFormat": {
-          "$ref": "fields#technicalFileFormatForAudio"
```

Comment:
This should include file formats for bin. I'll look through the media info code to see what values it can produce, but can you test a couple of BINs to see what the report format is?